addons/mail/static/src/core/common/composer.xml

```diff
@@ -13,6 +13,7 @@
                     'o-isUiSmall': ui.isSmall,
                     'p-3': normal,
                     'o-hasSelfAvatar': !env.inChatWindow and thread,
+                    'o-focused': props.composer.isFocused
```

Comment:
This has been added to help `contains()` observe the focus mutation: `:focus` is not a mutation, so this would rely on timeout of `contains()`, which waste 3 seconds.

Reply:
So this will add extra rendering when focusing, not the best way to go about it either.

Contains should register an even listener for focus when it waits for a focus selector instead, so all usages are automatically covered.

Reply:
And if we can have one for values of input too that would be perfect, as several tests are waiting on timeout for that. I never find time to take care of these.

Reply:
> Contains should register an event listener for focus when it waits for a focus selector instead, so all usages are automatically covered.

This would indeed be nice. Can be a bit tricky for contains to dissect the selector to know which HTML element to await `:focus`.